cmd/monaco/v2/deploy/deploy.go

```diff
@@ -87,7 +87,11 @@ func Deploy(fs afero.Fs, deploymentManifestPath string, specificEnvironment stri
 	}
 
 	if specificProject != "" {
-		filtered, err := filterProjectIdsByName(projects, []string{specificProject})
+		projectsMap := strings.Split(specificProject, ",")
```

Comment:
This is actually a slice and not a map. I would rather call this var `projectsToDeploy` or 
something like this.